src/main/java/Duke.java

```diff
@@ -1,10 +1,162 @@
+        printLine();
+    }
+    public static void sayBye() {
+        printLine();
+        System.out.println("\tBye. Hope to see you again soon!");
+        printLine();
+    }
+
+
+
+    final static int MAXTASKS = 100;
+    public static Task[] taskList = new Task[MAXTASKS];
+    public static int currentIndex = 0;
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("---------------------------------------------------------------------------------");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("---------------------------------------------------------------------------------");
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        while (true) { // ensure that the loop can stay on forever if needed.
+            userInput = in.nextLine();
+            if(userInput.equals("bye")) { // exit command
+                break;
+            } else if(userInput.equals("list")) { //displays the list if needed
+                printLine();
+                System.out.println("\tHere are the tasks in your list:");
+                listTasks(currentIndex, taskList);
+                printLine();
+            } else if (isTheSame(userInput, "mark")) { //mark the task in
+                while(isInRange(userInput, currentIndex)==false) {
+                    printLine();
+                    System.out.println("\tNice try, enter a valid index to mark:");
+                    printLine();
+                    userInput = in.nextLine();
+                }
+                printLine();
+                printMarkedTask(userInput, taskList);
+                printLine();
+            } else if (isTheSame(userInput, "unmark")) {//unmark the task
+                while(isInRange(userInput, currentIndex)==false) {
+                    printLine();
+                    System.out.println("\tNice try, enter a valid index to unmark:");
+                    printLine();
+                    userInput = in.nextLine();
+                }
+                printLine();
+                printUnmarkedTask(userInput, taskList);
+                printLine();
+            } else if(isTheSame(userInput, "todo")) {
+                printTodo(userInput);
+                //leave this for the final refactoring
+            } else if(isTheSame(userInput, "deadline")) {
+               printDeadline(userInput);
+            } else if(isTheSame(userInput, "event")) {
+                printEvent(userInput);
+            }
+            else { // tells the user that we have added the task in
+                printTask(userInput);
+            }
+        }
```

Comment:
Proper **if-else** coding standards are met according to the **Java Coding Standard**.